### PR TITLE
ACCount's mining update: bigger & better!

### PR DESCRIPTION
**Pull request by ACCount12**
My mining update, ported by @pudl and me.
- Plasma cutter is a gun now: it fires plasma bursts that can digs a line of tiles at once, consuming liquid plasma from it's internal storage.
- You can refill your plasma cutter with plasma sheets, plasma ore or liquid plasma.
- You can buy a plasma cutter in mining vendomat for points or for voucher.
- Advanced plasma cutter is avaible in RnD. It holds more plasma and deals more damage to asteroid, walls and mobs.
- Buffed drills a bit. Now they are 100% useless. They were 200% useless before.
- Increased advanced mining scanner's scanning speed.
- Decreased mesons and regular plasma cutter cost in RnD.
- You will see 1 tlie around you in dark with mesons, not 0.
- Ripley's armor is buffed a bit.
- Increased Ripley's tool slots: 6 for regular Ripley and 5 for firefighter modification.
- Mech plasma cutter added. Just like advanced plasma cutter, but for Ripley. Consumes power cell charge.
- Gibtonite will activate when hit with force => 10 items.
- You can defuse activated gibtonite with multitool.
- You can make a remote bomb out of gibtonite. Just attach igniter and signaller. Explosions are adminlogged.
- You can't hide a gibtonite bomb in lockers or backpacks. Gibtonite is still huge two-handed anchored item.
- ~~Telecrystals~~ Bluespace crystals added to asteroid. They are just as rare as diamonds, but there is only one crystal in each crystal tile.
- Moved all mining-related items sprites to /obj/mining.dmi from /obj/items.dmi.
- Moved all asteroid walls to /turf/mining.dmi from /turf/walls.dmi.
- Changed asteroid walls internal mechanics. They are a bit less hardcoded now.
- Added me to admins.txt. 

This pull replaces #8544.

### CODE REVIEW

**Comment by ACCount12** [comment]
Travis CI build failed, but only for Metastation. With a very strange log: https://travis-ci.org/tgstation/-tg-station/jobs/56189470

**Comment by ACCount12** [comment]
Yep, it was Travis CI's fault.

**Comment by Jalleo** [comment]
@ACCount12 for balance can i suggest you do the same thing for the normal gun with the mech if possible charge is a annoying thing when it runs out.

**Comment by ACCount12** [comment]
It works just like any other mech gun.

**Comment by Jalleo** [comment]
Other mech guns do have ammo as well. How long can the charge of a mech last while its mining if the plasma cutter is used extensively.

**Comment by ACCount12** [comment]
No, energy mech guns have no charges.

**Comment by Jalleo** [comment]
I was meaning the bullet ones but meh

**Comment by Lumipharon** [comment]
Can you please make drills not use power again? It was a awful, terrible idea to change this in the first place, and it's objectively, completely useless so long as it uses charge.

**Comment by Chocobro** [comment]
:+1::+1::+1: very good, thought out changes.

**Comment by bgobandit** [comment]
>Plasma cutter is a gun now: it fires plasma bursts that can digs a line of tiles at once, consuming liquid plasma from it's internal storage.

rip fighting blobs

**Comment by GunHog** [comment]
Is it possible to give the Ripley a scanner? That is the only annoyance left that would deter a miner wanting to use one. It is not possible to use items while in a mech, so a miner cannot find most minerals without having an activated advanced mining scanner with them.

**Comment by ACCount12** [comment]
My update had Advanced Meson Visor back on NT. Ripley had a meson visor with mining HUD and better night vision vars. Sadly, advanced mesons were lost in porting. Maybe i will add this back, but i think advanced mining scanner is not that costly. 400 points is two minutes of mining.

**Comment by ACCount12** [comment]
Merge now please. I want to make another pull, but item.dmi changes would conflict.

**Comment by stapler2025** [comment]
24 hour minimum wait sir.

**Comment by Nienhaus** [comment]
"Commits 19" Holy shit

**Comment by Kromgar** [comment]
Also is this  balanced for blob? Because it seems like Blobs would be royally fucked against these monstrosities

**Comment by ACCount12** [comment]
It's not worse than regular plasma cutter + laser. Laser holds 10x20=200 fire damage of lasers. Cutter holds 15x10=150 fire damage of lasers.

**Comment by GunHog** [comment]
And according to this, you can still *melee* using the cutters, so they <b>Are still the weapon of choice against blobs, behind the Xray gun</b>.

**Comment by GunHog** [comment]
Would you consider removing the power cell mechanic for drills? No miner seems to want them now, especially since the Kinetic Accelerator is currently the only tool a miner will ever need.

**Comment by ACCount12** [comment]
MERGE THIS PLZ
I already resolved one goddamn conflict, and i don't want to resolve another one.

**Comment by ACCount12** [comment]
Thanks.

File: config/admins.txt [hunk 1]
```diff
@@ -80,4 +80,4 @@ allura = Game Master
 lo6a4evskiy = Game Master
 vekter = Game Master
 Ahammer18 = Game Master
-Fayrik = Game Master
```
Comment (Fayrik) [inline config/admins.txt]: Oh that's where it went, I was looking for that.

**Pull request merged**